js/securesubmit/checkout-form.js

```diff
@@ -328,8 +328,8 @@ document.observe('dom:loaded', function () {
     };
     // Amasty completeCheckout();
 
-    if (typeof completeCheckout == 'function' && document.querySelector('#amscheckout-onepage')) {
-        amastyCompleteCheckoutOriginal = cloneFunction(completeCheckout);
+    if (typeof completeCheckout == 'function' && document.getElementById('amscheckout-onepage')) {
```

Comment:
@ttnnkkrr I didn't notice this before, but can you make the `==` to be `===`? Calls to `typeof` should be compared strictly